common/source/docs/common-scripting-step-by-step.rst

```diff
@@ -5,21 +5,36 @@
 Script Setup and Use Examples
 =============================
 
-Testing a Script in SITL
+Running a Script in SITL
 ========================
 In Linux:
 
-#. Assuming you have setup the build and simulation environment, run the simulation. See :ref:`using-sitl-for-ardupilot-testing`. For example:
+#. Assuming you have setup the build and SITL environment, run SITL. See :ref:`using-sitl-for-ardupilot-testing`. For example:
 
-.. code::
+.. code-block:: bash
 
-    sim_vehicle.py -v plane -f quadplane
+    cd /path/to/ardupilot
+    ./Tools/autotest/sim_vehicle.py -v plane --console
 
 Mavproxy ground control station will automatically be started. You can also start QGC (using the QGroundControl.AppImage) and/or Mission Planner (using WISE to run under Linux). Any or all three can be used simultaneously.
 
-#. From your GCS, enable scripting by setting the :ref:`SCR_ENABLE<SCR_ENABLE>` parameter to "1". Restart the simulation.
-#. A "scripts" sub-directory will be created in the directory you started the simulation, if it did not already exist. Place the script you downloaded into this directory. Again, restart the simulation.
-#. If you get a "Scripting: out of memory" GCS message then you will need to increase :ref:`SCR_HEAP_SIZE<SCR_HEAP_SIZE>` parameter and try to restart the simulation.
+#. From your GCS, enable scripting by setting the :ref:`SCR_ENABLE<SCR_ENABLE>` parameter to "1". Restart the simulation, or in MAVProxy, type ``reboot``.
```

Comment:
we say "restart the simulation" but never explain how, ie CRTL-C, re-enter run SITL command (up arrow on some systems)

Reply:
That should be covered in the SITL docs, but it doesn't look like it is yet.